src/js/components/inspectors/Property.tsx

```diff
@@ -71,12 +73,10 @@ function mapStateToProps(reduxState: State, ownProps: OwnProps): StateProps {
   const field = getIn(state, path + '.field');
   const scaleName = scale && getInVis(reduxState, 'scales.' + scale + '.name');
 
-  const value = getIn(state, path);
-
   return {
     group:  getIn(state, path + '.group'),
     signal: getIn(state, path + '.signal'),
-    value:  value !== null && value !== undefined ? value : ownProps.value,
+    value: ownProps.value === undefined ? getIn(state, path) : undefined, // Not using || because value could be 0
```

Comment:
observation: it's now no longer possible for the value of `ownProps.value` to be directly returned if it's not `undefined`. is that what we want?

Reply:
thnx for putting in the fix for this.